pom.xml

```diff
@@ -85,8 +85,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
-                    <source>8</source>
-                    <target>8</target>
+                    <source>${maven.compiler.source}</source>
```

Comment:
Isn't this a no-operation? 
`maven-compiler-plugin` uses `maven.compiler.source` as a user property for this config.
https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#target

I do want to state somewhere in the pom that the minimal compiler version is 8. Otherwise the current default of 1.7 will cause compilation errors.

Reply:
I understood, that's why you put `<source>8</source>`, but for that the right value is `<source>1.8</source>`; since you putted a corcert value line 35 & 36 I choosed to use it.

Reply:
Anyhowtheir is many ways to set a working config. 
I.E.:
```xml
            ...
            <properties>
                 <maven.compiler.source>1.8</maven.compiler.source>
                 <maven.compiler.target>1.8</ maven.compiler.target>
            </properties>
            ...
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.0</version>
                <configuration>
                    <source>${maven.compiler.source}</source>
                    <target>${maven.compiler.target}</target>
                </configuration>
            </plugin>
            ...
```
or
```xml
            ...
            <properties>
                 <maven.compiler.source>1.8</maven.compiler.source>
                 <maven.compiler.target>1.8</ maven.compiler.target>
            </properties>
            ...
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.0</version>
            </plugin>
            ...
```
or
```xml
            ...
            <properties>
                 <maven.compiler.source>1.8</maven.compiler.source>
                 <maven.compiler.target>1.8</ maven.compiler.target>
            </properties>
            ...
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.8.0</version>
                <configuration>
                    <source>${maven.compiler.source}</source>
                    <target>${maven.compiler.target}</target>
                </configuration>
            </plugin>
            ...
```

Reply:
Oh, I've missed that java version is both configured with props and plugin config. But with different values. The plugin config is unnecessary then and I'll remove it. I'll leave it with the shortest option:
```
            ...
            <properties>
                 <maven.compiler.source>1.8</maven.compiler.source>
                 <maven.compiler.target>1.8</maven.compiler.target>
            </properties>
            ...
```
Thank you!